src/storages/PerpStorage.sol

```diff
@@ -50,6 +50,8 @@ contract PerpStorage is OwnableUpgradeable, ReentrancyGuardUpgradeable, IPerpSto
   uint256 public movingWindowLength;
   uint256 public movingWindowInterval;
 
+  mapping(bytes32 positionId => uint256 lastIncreaseSize) public lastIncreaseSizeByPositionId;
```

Comment:
This can be added to `struct Position` itself, don't need a new slot.

Reply:
Added to Position struct